packages/ethereum-contracts/ops-scripts/deploy-deterministically.js

```diff
@@ -110,6 +110,9 @@ module.exports = eval(`(${S.toString()})()`)(async function (
         data: ContractArtifact.bytecode,
         arguments: deployArgs,
     });
+    if (process.env.GAS_PRICE !== undefined) {
```

Comment:
I missed this one, what do you use it for?

Reply:
to deploy the forwarders at the same address